tests/scripts/run_embedded_app.py

```diff
@@ -40,16 +40,36 @@ def launch_app(args):
         init_msg = "The app is initialized" if is_initialized() else "The app is not initialized"
         print(init_msg)
 
-    app = pymechanical.App(
-        version=int(args.version),
-        private_appdata=args.private_appdata,
-        copy_profile=False,
-        globals=globals(),
-    )
+    if args.no_globals_update:
+        app = pymechanical.App(
+            version=int(args.version),
+            private_appdata=args.private_appdata,
+            copy_profile=False,
+        )
+    else:
+        app = pymechanical.App(
+            version=int(args.version),
+            private_appdata=args.private_appdata,
+            copy_profile=False,
+            globals=globals(),
+        )
 
     return app
 
 
+def test_globals(args):
+    """Test ViewOrientationType doesn't throw an error when globals are updated."""
+    # Set BUILDING_GALLERY to True
+    pymechanical.BUILDING_GALLERY = True
+    # Launch an embedded instance of the app without updating globals
+    app = pymechanical.App(version=int(args.version))
+    # Launch an embedded instance of the app with updated globals
+    app = launch_app(args)
```

Comment:
it would be more readable here to do this:
```
app = pymechanical.App(version=int(args.version))
app = pymechanical.App(globals=globals())
```

rather that to reuse launch_app, since the other arguments are not related to this tet.